pkg/garden/app/treaty.hoon

```diff
@@ -105,6 +105,9 @@
     ?>  =(our.bowl src.bowl)
     =.  direct  (~(put in direct) [ship desk])
     :_(this (drop ~(safe-watch tr:cc [ship desk])))
+    ::
+      [%treaties ~]
+    `this
```

Comment:
If we give new/removed treaty updates on this path, we should also give an initial result here, right?